src/Altinn.Profile.Integrations/Migration/profiledb.sql

```diff
@@ -0,0 +1,54 @@
+﻿-- Drop the database if it exists
+DROP DATABASE IF EXISTS profiledb;
+
+-- Create the database
+CREATE DATABASE profiledb
+    WITH
+    OWNER = postgres
+    ENCODING = 'UTF8'
+    LC_COLLATE = 'Norwegian_Norway.1252'
+    LC_CTYPE = 'Norwegian_Norway.1252'
+    LOCALE_PROVIDER = 'libc'
+    TABLESPACE = pg_default
+    CONNECTION LIMIT = -1
+    IS_TEMPLATE = False;
+
+-- Create schema if it doesn't exist
+CREATE SCHEMA IF NOT EXISTS contact_and_reservation;
```

Comment:
We need statements that grants the two users we have access to the schema:

GRANT ALL ON SCHEMA contact_and_reservation TO platform_profile_admin;
GRANT USAGE ON SCHEMA contact_and_reservation TO platform_profile;